zapd/models.py

```diff
@@ -759,3 +823,34 @@ def is_accessible(self):
             self.can_export = True
             return True
 
+class WalletRestrictedModelView(sqla.ModelView):
```

Comment:
update this with a filter to search for wallets with more then one device

and make it visible via the flask admin UI

Reply:
just pulled the latest update.
Erroring at the moment when I try to go to the AM Wallet page:

Its refering to the amdevice_list.html

Currently this page doesn't appear to exists.